### ISSUE TITLE
When clicking on a link-chain button next to a header in a Markdown document, copy the URL automatically

### ISSUE THREAD

**Issue by leventov** [refined-github/refined-github#5131]
### Description

When one clicks on the "link-chain" button to the left of any header in a Markdown document, Github just navigates to this page section (the URL in browser window becomes README.md#section) but doesn't copy it into the clipboard automatically, as many modern web interfaces do in this situation, so I often forget to copy the URL from the browser and switch to another window to paste whatever was in my clipboard already.

### Example URL

https://github.com/refined-github/refined-github

**Comment by fregante** [comment]
This sounds similar to our `copy-on-y` feature (which I wish was available on more pages, by the way)

**Comment by kidonng** [comment]
Yeah the intent to copy a link should be very clear here. Maybe we can just turn them into `<clipboard-copy>` or something like that, along with a tooltip.

**Comment by fregante** [comment]
We can just use `delegate` + the toast to notify of the copy, so we don't have to change the DOM of every such link

**Comment by fregante** [comment]
Right click, copy URL. Closing 4-year-old request to save 1 click, as nice as it sounds
